papaparse.js

```diff
@@ -415,6 +415,24 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
```

Comment:
Why is this class required? I think we can directly manage the isIncluded function in our parser and this will simplify the code.